index.d.ts

```diff
@@ -50,6 +50,14 @@ export interface IMuiVirtualizedTableColumn<TRow = any> {
    */
   width?: number | string;
 
+  /**
+   * Minimum width of cell.
+   *
+   * Must be a `number`
+   */
+  minWidth?: number;
```

Comment:
Based on the implementation in utils, this can only be a number unlike the widths prop.